scripts/python-maintainer/makefile_template.jinja2

```diff
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:={{ openwrt_name }}
```

Comment:
I would like if you can a little bit reorder some things in Makefile:

```suggestion

PKG_NAME:={{ openwrt_name }}
PKG_VERSION:={{ version }}
PKG_RELEASE:=1

PKG_SOURCE:={{ filename }}
PKG_SOURCE_URL:=https://pypi.python.org/packages/source/{{ first_char }}/{{ package }}/
PKG_HASH:={{ sha256 }}

PKG_MAINTAINER:={{ maintainer }}
PKG_LICENSE:={{ license }}
PKG_LICENSE_FILES:=

PKG_BUILD_DIR:=$(BUILD_DIR)/{{ package }}-$(PKG_VERSION)
```

We should also forget about PKG_CPE_ID. 

BTW: PKG_LICENSE_FILES is not always matching LICENSE file, but there could be also COPYING and so on.